middle_end/flambda2/parser/flambda_to_fexpr.ml

```diff
@@ -672,7 +672,11 @@ let set_of_closures env sc =
       (fun (function_slot, fun_decl) ->
         function_declaration env fun_decl function_slot alloc)
       (Set_of_closures.function_decls sc
-      |> Function_declarations.funs_in_order |> Function_slot.Lmap.bindings)
+      |> Function_declarations.funs_in_order
+      |> Function_slot.Lmap.map (function
+           | Function_declarations.Deleted _ -> Misc.fatal_error "todo"
```

Comment:
is that a todo for another PR or for alter this PR ?

Reply:
Most likely for another PR, there are many other missing features in fexpr so it would make sense to make a single PR to add them all.